Silicon/MeteorlakePkg/Include/Library/IgdOpRegionLib.h

```diff
@@ -0,0 +1,40 @@
+/**@file
+  This is part of the implementation of an Intel Graphics drivers OpRegion /
+  Software SCI interface between system Bootloader, ASL code, and Graphics drivers.
+  The code in this file will load the driver and initialize the interface
+  Supporting Specification: OpRegion / Software SCI SPEC 0.70
+
+  Acronyms:
+    IGD:        Internal Graphics Device
+    NVS:        ACPI Non Volatile Storage
+    OpRegion:   ACPI Operational Region
+    VBT:        Video BIOS Table (OEM customizable data)
+    IPU:        Image Processing Unit
+
+  Copyright (c) 2021, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
```

Comment:
Please check if you could use common header file Silicon\CommonSocPkg\Include\Library\IgdOpRegionLib.h

Reply:
patch submitted with removed duplicate lib